Platform/ApollolakeBoardPkg/CfgData/CfgData_Ext_Gpmrb.dlt

```diff
@@ -1,10 +1,13 @@
-## @file
+#/** @file
+#
 #  Platform Configuration Delta File.
 #
-#  Copyright (c) 2018, Intel Corporation. All rights reserved.<BR>
+#  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
```

Comment:
Often we don't remove the old year while we extend Copyright. Better change to:
#  Copyright (c) 2018 - 2019, Intel Corporation. All rights reserved.<BR>

Reply:
This is been done by the Config Editor. We try not to overwrite the output of the tool.

Reply:
Thanks for explaining this. It is fine then.